src/test/java/org/opensearch/security/test/AbstractSecurityUnitTest.java

```diff
@@ -240,7 +240,9 @@ protected void initialize(ClusterInfo info, Settings initTransportClientSettings
             builder.putList("plugins.security.authcz.admin_dn", "CN=kirk,OU=client,O=client,l=tEst, C=De");
             builder.put(ConfigConstants.SECURITY_BACKGROUND_INIT_IF_SECURITYINDEX_NOT_EXIST, false);
         }
-
+        // disable disk threshold by default, this sometimes interferes with
```

Comment:
as this also isn't directly related to this PR this should IMHO be a dedicated PR or at the very least a dedicated commit

Reply:
Agreed, I will submit a separate PR for this.